### DISCUSSION TITLE
Could not receive any emails to aliases

### DISCUSSION THREAD

**Discussion by 3gmgithub** [simple-login/app#1527] [Self-hosting question]
I was having a hard time receiving emails to any alias.  However, the transactional emails (Invite, newsletter, help emails...) were all being sent without trouble.

Looks like spamhaus has an update for postfix that solved my issue.

Here is what I needed to update in the main.cf:

```
# Recipient restrictions:
smtpd_recipient_restrictions =
   reject_unauth_pipelining,
   reject_non_fqdn_recipient,
   reject_unknown_recipient_domain,
   permit_mynetworks,
   reject_unauth_destination,
#   reject_rbl_client zen.spamhaus.org,
#   reject_rbl_client bl.spamcop.net,
   permit

postscreen_dnsbl_sites = zen.spamhaus.org=127.0.0.[2..11]
```

The emails started working again!!!  However, I am wondering if there are better options I should choose.
Does anyone have any suggestions? Hopefully, this helps someone banging their head like I was!

**Comment by PerpetualNooby** [comment]
Hi, I am having the same situation, where can I find the main.cf file? I do not have my own server, is this kind of repair off the table for me?
